### PR TITLE
added custom fonts

### PR DESCRIPTION
**Pull request by mihirverma7781**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import './fonts/gilroy/Gilroy-Black.ttf';
+import './fonts/gilroy/Gilroy-Bold.ttf';
+import './fonts/gilroy/Gilroy-ExtraBold.ttf';
+import './fonts/gilroy/Gilroy-Heavy.ttf';
+import './fonts/gilroy/Gilroy-Light.ttf';
```
Comment (mihirverma7781) [inline src/index.js:9]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-Light.ttf";
```

File: src/index.js [hunk 2]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
...
+import './fonts/gilroy/Gilroy-Medium.ttf';
+import './fonts/gilroy/Gilroy-Regular.ttf';
+import './fonts/gilroy/Gilroy-SemiBold.ttf';
+import './fonts/gilroy/Gilroy-Thin.ttf';
+import './fonts/gilroy/Gilroy-UltraLight.ttf';
```
Comment (mihirverma7781) [inline src/index.js:14]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-UltraLight.ttf";
```

File: src/index.js [hunk 3]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import './fonts/gilroy/Gilroy-Black.ttf';
+import './fonts/gilroy/Gilroy-Bold.ttf';
+import './fonts/gilroy/Gilroy-ExtraBold.ttf';
```
Comment (mihirverma7781) [inline src/index.js:7]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-ExtraBold.ttf";
```

File: src/index.js [hunk 4]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
...
+import './fonts/gilroy/Gilroy-Heavy.ttf';
```
Comment (mihirverma7781) [inline src/index.js:8]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-Heavy.ttf";
```

File: src/index.js [hunk 5]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
...
+import './fonts/gilroy/Gilroy-Light.ttf';
+import './fonts/gilroy/Gilroy-Medium.ttf';
+import './fonts/gilroy/Gilroy-Regular.ttf';
```
Comment (mihirverma7781) [inline src/index.js:11]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-Regular.ttf";
```

File: src/index.js [hunk 6]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
...
+import './fonts/gilroy/Gilroy-SemiBold.ttf';
+import './fonts/gilroy/Gilroy-Thin.ttf';
```
Comment (mihirverma7781) [inline src/index.js:13]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-Thin.ttf";
```

File: src/index.js [hunk 7]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
...
 import './index.css';
 import App from './App';
+import './fonts/gilroy/Gilroy-Black.ttf';
+import './fonts/gilroy/Gilroy-Bold.ttf';
+import './fonts/gilroy/Gilroy-ExtraBold.ttf';
+import './fonts/gilroy/Gilroy-Heavy.ttf';
+import './fonts/gilroy/Gilroy-Light.ttf';
+import './fonts/gilroy/Gilroy-Medium.ttf';
+import './fonts/gilroy/Gilroy-Regular.ttf';
+import './fonts/gilroy/Gilroy-SemiBold.ttf';
```
Comment (mihirverma7781) [inline src/index.js:12]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-SemiBold.ttf";
```

File: src/index.js [hunk 8]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import './fonts/gilroy/Gilroy-Black.ttf';
```
Comment (mihirverma7781) [inline src/index.js:5]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-Black.ttf";
```

File: src/index.js [hunk 9]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
...
+import './fonts/gilroy/Gilroy-Bold.ttf';
```
Comment (mihirverma7781) [inline src/index.js:6]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-Bold.ttf";
```

File: src/index.js [hunk 10]
```diff
@@ -2,6 +2,16 @@ import React from 'react';
...
+import './fonts/gilroy/Gilroy-ExtraBold.ttf';
+import './fonts/gilroy/Gilroy-Heavy.ttf';
+import './fonts/gilroy/Gilroy-Light.ttf';
+import './fonts/gilroy/Gilroy-Medium.ttf';
```
Comment (mihirverma7781) [inline src/index.js:10]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/The-Major-Project/Design-and-Develop-frontend/pullRequest?prid=8041234)
```suggestion
import "./fonts/gilroy/Gilroy-Medium.ttf";
```

File: src/index.css [hunk 11]
```diff
@@ -1,4 +1,73 @@
...
+
+@font-face {
+  font-family: "glight";
+  src: local("glight"),
+    url("./fonts/gilroy/Gilroy-Light.ttf") format("truetype");
+  font-weight: 200;
+}
+
+@font-face {
+  font-family: "gthin";
```
Comment (yash-sharma01) [inline src/index.css:69]: It should be ultraLight

**Review by yash-sharma01 — CHANGES_REQUESTED** [review]
Change the name of last ultraLight font

**Review by yash-sharma01 — APPROVED** [review]

**Pull request merged**